exams/signals.py

```diff
@@ -27,12 +26,3 @@ def update_exam_authorization_final_grade(sender, instance, **kwargs):  # pylint
     """
     mmtrack = get_mmtrack(instance.user, instance.course_run.course.program)
     authorize_for_exam(mmtrack, instance.course_run)
-
-
-@receiver(post_save, sender=CachedEnrollment, dispatch_uid="update_exam_authorization_cached_enrollment")
-def update_exam_authorization_cached_enrollment(sender, instance, **kwargs):  # pylint: disable=unused-argument
```

Comment:
We still need to create an `ExamProfile` when the user first enrolls, although an `ExamAuthorization` should not be created at that point because they presumably shouldn't have passed yet.